### PR TITLE
Issue/14 - Implement purchases API

### LINKED ISSUE #14: Implement new purchases API
ConvertKit has a purchases API now to record purchase data on accounts. 

http://developers.convertkit.com/#purchases

### PR DESCRIPTION
**Pull request by nosegraze**
Closes #14

### CODE REVIEW

File: includes/class-edd-convertkit.php [hunk 1]
```diff
@@ -31,9 +38,11 @@ public function init() {
 			return;
 		}
 
-		$this->checkout_label = edd_get_option( 'edd_convertkit_label', __( 'Signup for the newsletter', 'edd-convertkit' ) );
+		add_filter( 'edd_use_after_payment_actions', '__return_false' );
```
Comment (pippinsplugins) [inline includes/class-edd-convertkit.php]: Make sure to remove this before merge :)

Comment (nosegraze) [reply includes/class-edd-convertkit.php]: Lol oops. Did you add that for testing?

Comment (pippinsplugins) [reply includes/class-edd-convertkit.php]: Yes I did.

**Comment by growdev** [comment]
Tags for purchases look good :+1:

convertkit 2018-07-03 10-14-45

Single and multiple products look good :+1:

convertkit 2018-07-03 10-12-17

Discounts are not sent, but I'm not sure if the discount total is stored in the payment?

convertkit 2018-07-03 10-12-40

If the discount total isn't sent can the code be added to the Order Details?

**Comment by nosegraze** [comment]
Thanks so much @growdev . I'll take a look at the discounts. :)

**Comment by nosegraze** [comment]
Updated. The discount amount should now go through.

**Comment by growdev** [comment]
The Discount line in ConvertKit is currently showing $229.60. At first I would expect it to be $98.40, or the amount the subtotal is reduced by to get the Total. 

convertkit 2018-07-03 13-03-31

**Comment by nosegraze** [comment]
Thank you @growdev .

I have made two small changes:

1. The discount amount should hopefully be correct now and show the amount that was removed off the price.
2. I also fixed the individual item amounts so that they don't include the discount amount. Those are now more like the subtotals for each item.

**Comment by growdev** [comment]
Looks great, thank you @nosegraze!

convertkit 2018-07-05 11-38-59

**Pull request merged**